cave_sea/map.conf

```
@@ -0,0 +1,18 @@
+team.2.color = blue
+team.1.pos = (-7,2,6)
+team.2 = blue
+team.1.color = red
+team.1 = red
+team.2.pos = (-7,2,-6)
+h = 150
+r = 75
+rotation = z
+author = sodomit
+name = Cave Sea
+base_node = ctf_map:glass
+initial_stuff = grenades:frag 1,default:mese_post_light 99
```

I don't think that this will be useful
```suggestion
initial_stuff = grenades:frag 1,default:pick_mese,default:torch 10
```

---

Maybe 20 or 20+ torches will be needed in case it's a cave.

---

default torches doesn't work in water.

---

Yes, I have made those changes but I don't think that we need too many mese postlights.

---

Why do you want to add mese pick to initial stuff?